debezium-testing/debezium-testing-openshift/src/test/java/io/debezium/testing/openshift/mysql/MySqlConnectorIT.java

```diff
@@ -138,4 +138,27 @@ public void shouldResumeStreamingAfterRedeployment() throws IOException, Interru
                 .untilAsserted(() -> assertRecordsCount(connectorName + ".inventory.customers", 6));
         assertRecordsContain(connectorName + ".inventory.customers", "jerry@test.com");
     }
+
+    @Test
+    @Order(7)
+    public void shouldBeDownAfterCrash() throws SQLException {
+        operatorController.disable();
```

Comment:
Should not this class have an `@AfterClass` method that would enable operator to make sure it is running for the further tests?

Reply:
Probably yes. However, I'd rather include this into @BeforeAll in order to minimize potential connector restarts.

Reply:
It's doing it a bit backwards -- the parent kinda presumes the child could break something. But in this case, I think ensuring that Operator runs before each test class is fair enough.